indy_node/server/node.py

```diff
@@ -333,7 +333,7 @@ def executeDomainTxns(self, ppTime, reqs: List[Request], stateRoot,
         :param ppTime: the time at which PRE-PREPARE was sent
         :param req: the client REQUEST
         """
-        return self.default_executer(DOMAIN_LEDGER_ID, ppTime, reqs,
+        return self.default_executor(DOMAIN_LEDGER_ID, ppTime, reqs,
```

Comment:
This is an API method defined in Plenum...
Please be careful when renaming API methods